README.md

```diff
@@ -82,9 +82,8 @@ About
 Curry is maintained and funded by thoughtbot, inc. The names and logos for
 thoughtbot are trademarks of thoughtbot, inc.
 
-We love open source software! See [our other projects][community] or look at
-our product [case studies] and [hire us][hire] to help build your iOS app.
+We love open source software! See [our other projects](https://thoughtbot.com/tools) or [see some case studies][case studies]. We also have [iOS learning resources at Upcase by thoughtbot](https://thoughtbot.com/upcase/ios).
```

Comment:
As an iOS developer, does the content on https://thoughtbot.com/upcase/ios still look valuable to you? If it's evergreen, it will be useful to someone who comes across it and linking to it from our iOS repos helps people find it.

If it's not valuable, let's work with the Upcase team to pull it down.

How actively we're producing new content on Upcase doesn't need to influence the decision.